plugins/confirm.cpp

```diff
@@ -612,15 +612,7 @@ DFhackCExport command_result plugin_init (color_ostream &out, vector <PluginComm
 
 DFhackCExport command_result plugin_enable (color_ostream &out, bool enable)
 {
-    if (is_enabled != enable)
-    {
-        for (auto c : confirmations)
-        {
-            if (!c.second->apply(enable))
-                return CR_FAILURE;
-        }
-        is_enabled = enable;
-    }
+    is_enabled = enable;
```

Comment:
Not sure if I should just check if `is_enabled != enabled` before setting it always, or only if we are doing something when the state changes. Maybe just check anyways?

Reply:
I think this is fine.